README.md

```diff
@@ -1 +1,12 @@
-# bitmap-transformer
\ No newline at end of file
+# bitmap-transformer
+
+This app is used to transform a bitmap image from the CLI. 
+To use it, run 
+**./gradlew run --args 'input output transform'**
```

Comment:
You can make your code look good in markdown by using backticks. You can `write code within a sentence,` or write
 ```
a block of code
```

Reply:
Implemented this for the next PR. TY!